build-tools/static/md/mkdocs/docs/EarlierEras.md

```diff
@@ -0,0 +1,81 @@
+---
+source_path: build-tools/static/md/common/src/EarlierEras.md
+---
+
+
+## Earlier Eras
+
+Many of the material in the Conway specification is brought from the spefication of
+earlier eras. In this section we provide an account of how the earlier specifications
+have been modified.
```

Comment:
I don't think is clear what is meant by "have been modified". Earlier specifications haven't been modified, but that some things from them have been refined/adapted/improved in subsequent eras (and thus in Conway they appear possibly modified)